src/main/java/graphql/annotations/processor/graphQLProcessors/GraphQLAnnotationsProcessor.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2016 Yurii Rashkovskii
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ */
+package graphql.annotations.processor.graphQLProcessors;
+
+public interface GraphQLAnnotationsProcessor {
```

Comment:
Why does the interface does not contains any method ? It's the only entry point we have in OSGi environment, and should give access to all methods to register new types

Reply:
ive added some methods to the interface but i didnt quite understand which methods you ment- because the register type methods were not in the interface even before the refactor.
Any way i dont understand OSGI so much so if you want to add support for this I would recommend that you create a new PR for that

Reply:
Previously GraphQLAnnotationsProcessor was the interface for all methods of GraphQLAnnotations - now I don't have an entry point to create a gql type from a class (something like getObject() ). It could be in another interface - or even directly in GraphQLAnnotations class if we exposes it directly without interface (which is also possible in OSGi, but not recommended) - but the method getObject() need to be available somewhere. Actually I could use directly getObject() from GraphQLObjectHandler (which by the way takes the container, so I'm also happy with it).
In all cases I will fix OSGi issues in another PR once this one will be merged - I don't see anything blocking right now.